src/services/hmwk-service.js

```diff
@@ -344,12 +344,19 @@ class HmwkService {
 
   // Upload a PDF to monday.com
```

Comment:
Add a comment here that `uploadPDF` will clear the previous file and is only able to upload exactly 1 PDF file.